core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -1355,11 +1389,10 @@ private String buildInOperator(int size) {
         if (size == 1) {
             return "(?)";
         }
-        StringBuilder builder = new StringBuilder("(");
-        for (int i = 0; i < size - 1; i++) {
-            builder.append("?,");
-        }
-        builder.append("?)");
+        StringBuilder builder =
+                new StringBuilder("(")
+                        .append(StringUtils.join(Collections.nCopies(size, "?"), ","))
```

Comment:
Done. Thanks, I am new to Android.

BTW, are there any plans to bump minSdkVersion? API level 10 is 7 years old...